codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -925,5 +920,306 @@ final void incrementStreamableBytesForTree(int numBytes) {
          * Indicates whether or not there are frames in the pending queue.
          */
         abstract boolean hasFrame();
+
+        /**
+         * Returns the parameter from the last call to {@link #markWritabilty(boolean)} (if supported).
+         * {@code false} if not supported.
+         */
+        abstract boolean markWritabilty();
+
+        /**
+         * Save the state of writability (if this method is supported).
+         */
+        abstract void markWritabilty(boolean isWritable);
+    }
+
+    /**
+     * All events which may impact the writability of a {@link Http2Stream} go through this interface.
+     * Implementations can choose how to calculate writability or ignore it all together.
+     */
+    private interface WritabilityMonitor {
+        void pendingBytesChanged(int delta);
+
+        void incrementWindowSize(AbstractState state, int delta) throws Http2Exception;
```

Comment:
Thanks for pointing out the inconsistency. It seems like we use the `increment` terminology more consistently. I think I prefer to change `pendingBytesChanged` to `incrementPendingBytes`. WDYT?